js/views/chatAppVw.js

```diff
@@ -0,0 +1,267 @@
+              }
+            }
+            chat.set('avatarURL', model.get('serverUrl') + "get_image?hash=" + chat.get('image_hash') + "&guid=" + chat.get('guid'));
+            self.renderChat(chat);
+          });
+          $('#chatHeads').html(self.listWrapper);
+        }
+      }
+    });
+
+    this.listenTo(window.obEventBus, "socketMessageRecived", function(response){
+      this.handleSocketMessage(response);
+    });
+
+    this.listenTo(window.obEventBus, "openChat", function(guid, key) {
+      this.openChat(guid, key);
+    });
+
+  },
+
+  render: function(){
+    "use strict";
+    var self = this;
+
+    loadTemplate('./js/templates/chatApp.html', function(loadedTemplate) {
+      self.$el.html(loadedTemplate());
+    });
+
+    return this;
+  },
+
+  renderChat: function(model){
+    var chat = new chatView({
+      model: model
+    });
+    this.subViews.push(chat);
+    this.listWrapper.prepend(chat.el);
+  },
+
+  renderChatMessage: function(model){
+    var chatMessage = new chatMessageView({
+      model: model
+    });
+    this.subViewsChat.push(chatMessage);
+    this.listWrapperChat.prepend(chatMessage.el);
+  },
+
+  renderNoneFound: function(){
+    // Decide what to do here
+  },
+
+  refreshConversations: function() {
+
+  },
+
+  openChat: function(guid, key) {
+    var self = this;
+    this.newChat();
+    $('#inputConversationRecipient').val(guid);
+    $('#inputConversationKey').val(key);
+    $('#inputConversationMessage').focus();
+
+    // Load conversation from DB
+    this.chatMessages = new chatMessageCollection();
+    this.chatMessages.url = this.options.model.get('serverUrl') + "get_chat_messages?guid=" + guid;
+    var model = this.options.model;
+
+    this.listWrapperChat = $('<div class="border0 custCol-border-secondary flexRow"></div>');
+
+    this.chatMessages.fetch({
+      success: function(chatMessages, response) {
+        if(chatMessages.models.length < 1) {
+          console.log('none found');
+        } else {
+          __.each(chatMessages.models, function (chatMessage) {
+            "use strict";
+            if(chatMessage.image_hash === undefined) {
+              var hash = window.localStorage.getItem("avatar_" + chatMessage.get('guid'));
+              if(hash !== "") {
+                chatMessage.set('image_hash', hash);
+              }
+            }
+            if(chatMessage.get('outgoing')) {
+              chatMessage.set('avatarURL', model.get('serverUrl') + "get_image?hash=" + model.get('avatar_hash'));
+            } else {
+              chatMessage.set('avatarURL', model.get('serverUrl') + "get_image?hash=" + chatMessage.get('image_hash') + "&guid=" + chatMessage.get('guid'));
+            }
+            self.renderChatMessage(chatMessage);
+          });
+
+          $('#chatConversation .chatConversationContent').html(self.listWrapperChat).animate({ scrollTop: $('#chatConversation .chatConversationContent').prop("scrollHeight")}, 100);
```

Comment:
That's 100px. I looked at the existing messages, and noticed most were between 50 and 80 px tall. Since we can't get the height of the list of messages, instead I am multiplying the number of messages by 100px.

It doesn't hurt anything to set scrollTo to a number higher than it needs to be, so it might be good to multiply by 200 instead, just in case someone has very tall messages.